filesystems/loopdev/sanity/README.md

```diff
@@ -0,0 +1,13 @@
+# loopdevie sanity test suite
+Sanity test suite for loop device. This test  will mount various filesystems, and stress them through iozone. \
+Test Maintainer: [Xiong Zhou](mailto:xzhou@redhat.com)
+
+## How to run it
+
+### Dependencies
+Please refer to the top-level README.md for common dependencies. Test-specific dependencies will automatically be installed when executing 'make run'. For a complete detail, see PURPOSE file
```

Comment:
Missing full stop at the end of the last sentence 😅

Reply:
samething. :sweat_smile: